core/templates/pages/release-coordinator-page/components/alert-dialog.component.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for alerting the user about an error that has
+ * occurred.
+ */
+
+import { Component, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+@Component({
+  selector: 'alert-dialog',
+  templateUrl: './alert-dialog.component.html',
+})
+export class AlertDialogComponent {
+  constructor(@Inject(MAT_DIALOG_DATA) public error: Error) {}
+}
```

Comment:
Can't we use the existing (base) dialogs for the behaviour of this dialog? @srijanreddy98 or @nithusha21 might be able to help.

Reply:
I guess Brian just stuck with Angular Material for the entire page. This is not a user facing website I guess its fine... but incase we do decide to change this, here is an example: https://github.com/srijanreddy98/oppia/blob/develop/extensions/rich_text_components/Skillreview/directives/oppia-noninteractive-skillreview-concept-card-modal.component.ts#L27

Reply:
I ended up scrapping this for our existing `alerts.service.ts` logic, FYI